src/site/articles/doc-comment-guidelines/index.markdown

```diff
@@ -58,9 +64,9 @@ And one for a variable or property:
 const double pi = 3.1415926535897932;
 {% endprettify %}<!-- END(PI) -->
 
-For an example of a class description with example code, see Completer
-([generated doc](http://api.dartlang.org/dart_async/Completer.html),
-[source code](http://code.google.com/p/dart/source/browse/branches/bleeding_edge/dart/sdk/lib/async/future.dart)).
+For an example of a library with doc comments, see the `path` library.
+([generated doc](http://www.dartdocs.org/documentation/path/1.3.3/index.html),
+[source code](https://github.com/dart-lang/path/blob/1.3.3/lib/path.dart)).
```

Comment:
It looks like the source for the linked-to page is actually this (which surprised me): https://raw.githubusercontent.com/dart-lang/path/1.3.3/README.md